Crashlytics/Crashlytics/Components/FIRCLSApplication.m

```diff
@@ -44,6 +44,8 @@
   return @"mac";
 #elif TARGET_OS_TV
   return @"tvos";
+#elif TARGET_OS_WATCH
+  return @"ios"; // TODO: temporarily use iOS until Firebase can add watchos to the backend
```

Comment:
I suspect an extra space is needed before the comment to resolve the clang-format issue.  See https://github.com/firebase/firebase-ios-sdk#code-formatting